apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -143,59 +132,51 @@ type SelectAppsFormValues = {
     validBefore: string
 }
 
-const HiddenBlock = styled.div<{ hide?: boolean }>`
+export type SharingAppValues = {
```

Comment:
SharingAppValuesType

Reply:
Why do you need it here? You have already declared this exact variable in InputStep